torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -43,7 +43,8 @@ def horizontal_flip_image_tensor(image: torch.Tensor) -> torch.Tensor:
     return image.flip(-1)
 
 
-horizontal_flip_image_pil = _FP.hflip
+def horizontal_flip_image_pil(image: PIL.Image.Image) -> PIL.Image.Image:
```

Comment:
Without this thin wrapper, the dispatch test fails, because `horizontal_flip_image_pil` is never called. Only `_FP.hflip` is called.

The old test handle this by allowing the user to set another name for mocking, but I don't want to bring this complexity to the new tests. Since we need to have v2 "standalone" from v1 at some point anyway, might as well do it here now.

Reply:
I'm a bit concerned in general about modifying the code only to make tests easier. I guess that is OK in this case because it doesn't add much complexity to the code, but let's keep an eye on this

Reply:
Indeed. The issue here is that in the beginning of the v2 transform kernels, we had a lot of them just aliased to their v1 equivalent. The "old" v2 test framework accounted for that with

https://github.com/pytorch/vision/blob/d814772ebce5fa5575ba11a908c515abd0d17d6b/test/transforms_v2_kernel_infos.py#L42-L44

e.g.

https://github.com/pytorch/vision/blob/d814772ebce5fa5575ba11a908c515abd0d17d6b/test/transforms_v2_kernel_infos.py#L215-L217

The one above is obsolete now, since we removed the aliasing some time ago in #6983.

So basically here I'm just doing something we already planned to do in the first place, just a little earlier.